lib/streamlit/config.py

```diff
@@ -761,6 +761,16 @@ def _s3_secret_access_key():
     description="Theme name displayed in the UI for theme selection.",
 )
 
+_create_option(
+    "customTheme.setAsDefault",
+    description="""
+        Whether to set the custom theme to be the default theme for this
+        streamlit app if one is defined.
+        """,
+    default_val=True,
+    type_=bool,
+)
```

Comment:
Ah I was thinking we could just set the theme if it exists without a config variable. But including this and defaulting to true does the same! Whether or not we want to expose this to developers I leave up to product @asaini